test_collections/sdk_tests/support/yaml_tests/sdk_yaml_tests.py

```diff
@@ -108,10 +112,35 @@ def sdk_yaml_test_collection(
 
     files = yaml_test_folder.yaml_file_paths()
     version = yaml_test_folder.version
-    suites = _parse_all_sdk_yaml(yaml_files=files, yaml_version=version)
+    suites = _parse_all_yaml(yaml_files=files, yaml_version=version)
 
     for suite in suites:
         suite.sort_test_cases()
         collection.add_test_suite(suite)
 
     return collection
+
+
+def custom_yaml_test_collection(
+    yaml_test_folder: YamlTestFolder = CUSTOM_YAML_TEST_FOLDER,
+) -> YamlCollectionDeclaration:
+    """Declare a new collection of test suites with the 3 test suites."""
+    collection = YamlCollectionDeclaration(
+        name="Custom YAML Tests", folder=yaml_test_folder
+    )
+
+    files = yaml_test_folder.yaml_file_paths()
+    suites = _parse_all_yaml(yaml_files=files, yaml_version=None)
+
+    has_valid_test_case = False
+    for suite in suites:
+        if not suite.test_cases:
+            continue
+        has_valid_test_case = True
+        suite.sort_test_cases()
+        collection.add_test_suite(suite)
+
+    if not has_valid_test_case:
+        return None
```

Comment:
I believe this `has_valid_test_case` variable is not needed. You can just check if `collection.test_suites` is empty or not.

Reply:
Agreed!